source/config.json

```diff
@@ -5,8 +5,8 @@
   "license": "Copyright (c) 2014 upfront GmbH, all rights reserved",
 
   "assets": {
-    "css": ["./timeline.css", "./assets/stylesheets/slippry.css"],
-    "js": ["./assets/scripts/full_height.js", "./assets/scripts/jquery-2.1.3.min.js", "./assets/scripts/slippry.min.js", "./assets/scripts/activate_gallery.js"]
+    "css": ["./assets/stylesheets/slippry.css"],
```

Comment:
It would be nice if we don't bundle css in js files, so they are actually usable in production.
@gabrielhase do we plan to continue with this design?
This got replaced with the magazine design, right?

Reply:
We are moving this design to the magazine boilerplate, correct. Will take some time though.